data-stores/README.md

```diff
@@ -0,0 +1,51 @@
+# Why
+
+At the moment (Nov 19) we use Redis as a Celery backend (i.e. for storing results of Celery tasks) as well as a data store where we track copr builds, github app installations and whitelist of users.
```

Comment:
```suggestion
At the moment (Oct 19) we use Redis as a Celery backend (i.e. for storing results of Celery tasks) as well as a data store where we track copr builds, github app installations and whitelist of users.
```
Are you time-travelling?

Reply:
No, I'm not, are you? My desktop says Mon Nov 4

Reply:
It will be valid tomorrow!